client/webserver/site/src/js/markets.js

```diff
@@ -185,6 +185,7 @@ export default class MarketsPage extends BasePage {
     bind(page.mktBuyField, 'change', () => { this.marketBuyChanged() })
     bind(page.mktBuyField, 'keyup', () => { this.marketBuyChanged() })
     bind(page.rateField, 'change', () => { this.rateFieldChanged() })
+    bind(page.rateField, 'keyup', () => { this.rateFieldChanged() })
```

Comment:
We cannot run `rateFieldChanged` on `keyup` since `rateFieldChanged` manipulates the input to account for the rate step. Trying to input certain values would be impossible now, because it keeps changing as you type.